src/config/constants.ts

```diff
@@ -119,3 +120,18 @@ export const SEARCH_FORM_OPTIONS = {
     placeholder: `${example} Ornithology or Greek Architecture`,
   },
 }
+
+export const NYPL_LOCATIONS = {
```

Comment:
Right now in[ NYPL core,](https://github.com/NYPL/nypl-core/blob/master/vocabularies/csv/locations.csv) both schomburg and schwarzman labels already match this pattern (ie they start `{shortName}-`). For LPA locations, they are all prefixed with "Performing Arts Research Collections", which could be updated to "Library for the Performing Arts". I'm wondering if it's worth re-thinking this whole business with the mapping and the slugs. One alternate solution might be:
We could update the LPA locations with this short name in the pref label, and use the a substring of the preflabel to map to the address. 
We could even return the address in the discovery API response, though that would require a little more time to update the API responses. Updating NYPL core with a new LPA label would be pretty trivial, and would not delay this work at all. I think this would cut down a lot of the complexity of this area of the code.

@nonword do you have thoughts on this?

Reply:
The primary purpose of `location.prefLabel` in NYPL-Core is display of location labels in RC (via the discovery-api), so the question becomes: Do we want to represent LPA locations as "Performing Arts Research Collections" in some RC contexts (Search Results, Item Table) but call it "Library for the Performing Arts" on the Hold Request page? It does seem to me these should be unified, but I'm not sure which should be updated. It looks like DFE has used the "Library for the Performing Arts" delivery label for the past 8 years, so perhaps that's the old one and we can retire it. That would allow these "shortName" values to be derived from `location.prefLabel` in the delivery locations response. The address may have to continue to be hardcoded for now since the best source for that is the location-service, but the addresses there slightly disagree with the addresses shown here and we probably don't need that level of data-driven brittleness for something that never changes

Reply:
Sean says to go ahead with it as is, and we'll return to this convo after conferring comms. 
@dgcohen I'm wondering if the code can be updated to only update LPA prefLabels? It could act as inline documentation of the special case of LPA prefLabels requiring transformation when SASB and schomburg do not.

Reply:
Sounds good, i added a ternary that makes non-LPA locations default to the prefLabel. Let me know if that's not the desired behavior.